nautobot_device_lifecycle_mgmt/jobs/lifecycle_reporting.py

```diff
@@ -43,7 +43,10 @@ def test_device_software_validity(self) -> None:
             validate_obj.last_run = job_run_time
             validate_obj.run_type = choices.ReportRunTypeChoices.REPORT_FULL_RUN
             validate_obj.validated_save()
-        self.log_success(message=f"Performed validation on: {devices.count()} devices.")
+        self.logger.info(
+            "Performed validation on: %d devices."  # pylint: disable=logging-not-lazy, consider-using-f-string
+            % devices.count()
```

Comment:
The reason pylint is complaining is because this statement should be:
```py
        self.logger.info("Performed validation on: %d devices.", devices.count())
```
That way the string doesn't get interpreted until the logger needs to use it (if the logger level is set higher than `info` it won't get emitted so doesn't need to be interpreted)